tests/e2e/backup_restore_suite_test.go

```diff
@@ -56,245 +56,235 @@ func mysqlReady(preBackupState bool, twoVol bool, backupRestoreType BackupRestor
 	})
 }
 
-var _ = Describe("AWS backup restore tests", func() {
+type BackupRestoreCase struct {
+	ApplicationTemplate          string
+	PvcSuffixName                string
+	ApplicationNamespace         string
+	Name                         string
+	BackupRestoreType            BackupRestoreType
+	PreBackupVerify              VerificationFunction
+	PostRestoreVerify            VerificationFunction
+	AppReadyDelay                time.Duration
+	MustGatherFiles              []string            // list of files expected in must-gather under quay.io.../clusters/clustername/... ie. "namespaces/openshift-adp/oadp.openshift.io/dpa-ts-example-velero/ts-example-velero.yml"
+	MustGatherValidationFunction *func(string) error // validation function for must-gather where string parameter is the path to "quay.io.../clusters/clustername/"
+}
 
-	var _ = BeforeEach(func() {
-		testSuiteInstanceName := "ts-" + instanceName
-		dpaCR.Name = testSuiteInstanceName
-	})
+func runBackupAndRestore(brCase BackupRestoreCase, expectedErr error, updateLastBRcase func(brCase BackupRestoreCase), updateLastInstallTime func()) {
+	updateLastBRcase(brCase)
 
-	type BackupRestoreCase struct {
-		ApplicationTemplate  string
-		PvcSuffixName        string
-		ApplicationNamespace string
-		Name                 string
-		BackupRestoreType    BackupRestoreType
-		PreBackupVerify      VerificationFunction
-		PostRestoreVerify    VerificationFunction
-		AppReadyDelay        time.Duration
-		MaxK8SVersion        *K8sVersion
-		MinK8SVersion        *K8sVersion
```

Comment:
These two are removed from struct. I would add this back so don't have to reengineer this in the future

Reply:
I think this is still used in older versions of OCP (like 4.9 in 1.1 branch). But I will not cherry-pick these changes to any branch, so I think its safe to remove from master (which lowest OCP version is 4.11/4.12). But I can add back as well

Reply:
Safe to remove yup.. and yeah can always look to prior commits to add it back in the future if needed.. resolving.